src/cpp/session/SessionClientInit.cpp

```diff
@@ -380,6 +380,8 @@ void handleClientInit(const boost::function<void()>& initFunction,
 
    sessionInfo["session_id"] = module_context::activeSession().id();
 
+   sessionInfo["drivers_support_licensing"] = false;
```

Comment:
Much better! I sent a separate PR to PRO that I can close now that was making this boolean change. Will update this one and merge...